storage/api/Storage.Samples/DownloadByteRange.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_download_byte_range]
+
+using Google.Apis.Storage.v1;
+using Google.Cloud.Storage.V1;
+using System;
+using System.IO;
+using System.Net.Http;
+
+public class DownloadByteRangeSample
+{
+    /// <summary>
+    /// Downloads a byte range to local path.
+    /// </summary>
+    /// <param name="bucketName">The name of the bucket.</param>
+    /// <param name="objectName">The name of the object.</param>
+    /// <param name="firstByte">The first byte of the object content.</param>
+    /// <param name="lastByte">The last byte of the object content.</param>
+    /// <param name="localPath">Local Path where file will be downloaded.</param>
+    public void DownloadByteRange(
```

Comment:
```suggestion
    public async Task DownloadByteRangeAsync(
```
... and also update the name of the class, file and test class and file.